fapi-client/src/main/scala/com/gu/facia/api/contentapi/ContentApi.scala

```diff
@@ -17,22 +17,18 @@ object ContentApi {
   type AdjustSearchQuery = SearchQuery => SearchQuery
   type AdjustItemQuery = ItemQuery => ItemQuery
 
-  def buildHydrateQueries(client: ContentApiClientLogic, ids: List[String], adjustSearchQuery: AdjustSearchQuery = identity): Try[Seq[SearchQuery]] = {
+  def buildHydrateQueries(client: ContentApiClientLogic, ids: List[String], adjustSearchQuery: AdjustSearchQuery = identity): Response[Seq[SearchQuery]] = {
     def queryForIds(ids: Seq[String]) = adjustSearchQuery(client.search
       .ids(ids mkString ",")
       .pageSize(ids.size)
       .showFields("internalContentCode"))
 
-    Try {
-      IdsSearchQueries.makeBatches(ids)(ids => client.getUrl(queryForIds(ids))) match {
+    IdsSearchQueries.makeBatches(ids)(ids => client.getUrl(queryForIds(ids))) match {
```

Comment:
Yeah, was thinking that myself. I'm not sure, I don't think to. @robertberry is a fan of `Exceptions`. What do you think @robertberry ?